monai/bundle/scripts.py

```diff
@@ -1318,17 +1316,23 @@ def onnx_export(
         input_shape_ = _get_fake_input_shape(parser=parser)
 
     inputs_ = [torch.rand(input_shape_)]
-    net = parser.get_parsed_content(net_id_)
-    if has_ignite:
-        # here we use ignite Checkpoint to support nested weights and be compatible with MONAI CheckpointSaver
-        Checkpoint.load_objects(to_load={key_in_ckpt_: net}, checkpoint=ckpt_file_)
-    else:
-        ckpt = torch.load(ckpt_file_)
-        copy_model_state(dst=net, src=ckpt if key_in_ckpt_ == "" else ckpt[key_in_ckpt_])
 
     converter_kwargs_.update({"inputs": inputs_, "use_trace": use_trace_})
-    onnx_model = convert_to_onnx(model=net, **converter_kwargs_)
-    onnx.save(onnx_model, filepath_)
+
+    def save_onnx(onnx_obj: Any, filename_prefix_or_stream: str | IO[Any], **kwargs: Any) -> None:
```

Comment:
In this version, the `_export` function only supports exporting as files. I am not sure why the `| IO[Any]` is added here. If this is for exporting to IO, please also add this to the `_export` function and implement the IO exporting in it. Otherwise, I think we should remove this type hints.

Thanks  
Bin

Reply:
You are right. The `save_onnx` function here in `onnx_export` focuses only on saving the ONNX model. It handles file paths instead of `IO` objects, so it can be removed to avoid confusion.